mobify/capturejs
# WIP: get testing and CI running easily

## Changes
- upgrade dependencies which have become busted (grunt-conbtrib-qunit and grunt-saucelabs)
- make bower, browserify and grunt-cli dependencies explicit
- Make tests run and pass in circle ci 2.0
- Make tests run and pass and saucelabs on browsers they still support
-Some small README updates to match

## Steps to Test
- Verify that instructions in readme work for installation
- versify CI tests are passing
- verify saucelabs tests can be run and pass
- verify that README makes sense and can be followed
